src/components/events/partials/EventsPresentersCell.tsx

```diff
@@ -25,6 +25,7 @@ const EventsPresentersCell = ({
 			({ name }) => name === "presentersBibliographic"
 		);
 		if (!!filter) {
+      dispatch(goToPage(0));
 			await dispatch(editFilterValue({filterName: filter.name, value: presenter}));
```

Comment:
Fix the indentation:

```suggestion
			dispatch(goToPage(0));
			await dispatch(editFilterValue({filterName: filter.name, value: presenter}));
```